cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js

```diff
@@ -167,7 +166,7 @@ describe('ui-circulation-settings: Fixed due date schedules', () => {
       checkInDate: moment.utc().format(),
     })
       .then(() => {
-        Users.deleteViaApi(userId);
+        Users.deleteViaApi(userData.userId);
```

Comment:
Seems a lot of 'user' word here
`userData.userId`
let's use just userData.id

Reply:
done